source/command.txt

```diff
@@ -28,18 +42,17 @@ By default, ``runCommand`` returns an object of type
 can specify a return type for ``runCommand()`` as an optional second
 parameter.
 
-Example
--------
+Run Command Example: Full File
+------------------------------
 
 In the following sample code, we send the ``dbStats`` command to request
 statistics from a specific MongoDB database.
 
 .. include:: /includes/connect-guide-note.rst
 
-.. literalinclude:: /includes/usage-examples/code-snippets/Command.java
+.. literalinclude:: /includes/Command.java
   :language: java
```

Comment:
I: there is not code rendering, check the parser log for info 
also, spacing issue
```suggestion
.. literalinclude:: /includes/Command.java
   :language: java
```